tests/ui_tests/app/page/cluster/page.py

```diff
@@ -211,6 +220,97 @@ class ClusterConfigPage(ClusterPageMixin):
     """Cluster page config menu"""
 
     MENU_SUFFIX = 'config'
+    MAIN_ELEMENTS = [
+        ObjectPageLocators.title,
+        ObjectPageLocators.subtitle,
+        ClusterMainLocators.text,
+        ClusterConfigLocators.description_input,
+        ClusterConfigLocators.search_input,
+        ClusterConfigLocators.advanced_chbx,
+        ClusterConfigLocators.save_btn,
+        ClusterConfigLocators.history_btn,
+    ]
+
+    def fill_search_input(self, text: str):
+        self.find_element(ClusterConfigLocators.search_input).clear()
+        self.send_text_to_element(ClusterConfigLocators.search_input, text)
+
+    def click_save_btn(self):
+        self.find_and_click(ClusterConfigLocators.save_btn)
+
+    def click_history_btn(self):
+        self.find_and_click(ClusterConfigLocators.history_btn)
+
+    def select_comparing_history_by_index(self, index_from: int, index_to: int):
+        self.click_history_btn()
+        self.wait_element_visible(ClusterConfigLocators.HistoryRow.history_select).click()
+
+        def click_option(index):
+            self.wait_element_visible(ClusterConfigLocators.HistoryRow.option)
+            self.find_elements(ClusterConfigLocators.HistoryRow.option)[index].click()
+
+        click_option(index_from)
+        self.find_and_click(ClusterConfigLocators.HistoryRow.compare_select)
+        click_option(index_to)
+
+    @allure.step("Save config with name {config_name}")
+    def save_new_config_name(self, config_name: str):
+        self.find_element(ClusterConfigLocators.description_input).clear()
+        self.send_text_to_element(ClusterConfigLocators.description_input, config_name)
+        self.click_save_btn()
+
+    def clear_search_input(self):
+        self.find_and_click(ClusterConfigLocators.search_input_clear_btn)
+
+    def get_all_config_rows(self):
+        return [r for r in self.find_elements(ClusterConfigLocators.row) if r.is_displayed()]
+
+    def get_all_config_groups(self):
+        return self.find_elements(ClusterConfigLocators.group)
+
+    @contextmanager
+    def wait_rows_change(self):
+        """Wait changing rows amount."""
+
+        current_amount = len(self.get_all_config_rows())
+        yield
+
+        def wait_scroll():
+            assert (
+                len(self.get_all_config_rows()) != current_amount
+            ), "Amount of rows on the page hasn't changed"
+
+        wait_until_step_succeeds(wait_scroll, period=1, timeout=10)
```

Comment:
I see this manager is defined more than in one place, so I suggest we'll think of making in a utility function that takes the "get_rows" function, so we won't duplicate code

Reply:
Or we define such a manager in `BasePage` and somehow provide row_count function to it. Or make a mixin that we can add to "table like" classes.

Reply:
moved to configuration page